src/components/organisms/AppBar.vue

```diff
@@ -104,26 +120,27 @@
 </template>
 
 <script setup lang="ts">
-import { computed } from "vue"
+import { computed, ref } from "vue"
 import { useRoute, useRouter } from "vue-router"
-import { useDark, useToggle } from "@vueuse/core"
 import { useAuthStore } from "@/store/auth"
 import { Menu, MenuButton, MenuItem, MenuItems } from "@headlessui/vue"
 import { ArrowCircleDownIcon,ArrowCircleUpIcon, MoonIcon, SunIcon } from "@heroicons/vue/outline"
 import { usePodcastStore } from "@/store/podcasts"
+import { useDark, useToggle } from "@vueuse/core"
 
 const authStore = useAuthStore()
 const route = useRoute()
 const router = useRouter()
 const store = usePodcastStore()
-
+const todayDate = new Date(new Date().getTime() + 24 * 60 * 60 * 1000)
+const date = ref(todayDate.toISOString().substr(0, 10))
```

Comment:
1. you're calling the variable todayDate but its actually tomorrow
2. defining a date by adding some milliseconds seconds is ugly and theres weird edge cases where this won't work
3. theres no inherent need to store timestamps in the db, it'd be way more readable and easier to debug if you instead used yyyymmdd
4. substringing a number is dangerous, safer would be divide by a number and `math.floor` it

could/should be made a lot easier to read by splitting over a few lines and avoiding so many `new Date` but this is safer: `new Date(new Date().setDate((new Date()).getDate() + 1)).toISOString().split('T')[0]`